ci/azure-pipelines.yml

```diff
@@ -122,10 +134,21 @@ stages:
         - job: fvt
           displayName: 'FV Tests'
           dependsOn: main
+          variables:
+            VERSION: $[ dependencies.displayenv.outputs['BuildData.PACKAGE_VERSION'] ]              
           steps:
             - task: NodeTool@0
               inputs:
                 versionSpec: $(node_version_spec)
+            - script: |
+                set -ev 
+                export RELEASE_VERSION=${VERSION}-unstable.${COMMIT_ID:0:7}
+                find . \(-wholename "./apis/**/package.json" -or -wholename "./libraries/**/package.json" -or -wholename "./test/**/package.json" \) -exec node ci/updatePackageJson.js {} \;
+                node common/scripts/install-run-rush.js update
+              env:
+                COMMIT_ID: $(Build.SourceVersion)
+                VERSION: $(VERSION)                           
+              condition: and(succeeded(), eq(variables['Build.Reason'], 'Schedule'))  # only run on the scheduled builds                
```

Comment:
This seems to happen a few times- it would be good to use a template instead, like we do with `templates/build-data.yaml`